### ISSUE TITLE
The Demo at http://www.liedman.net/leaflet-realtime isn't working for me.

### ISSUE THREAD

**Issue by borosilicate** [perliedman/leaflet-realtime#188]
The site the realtime data is supposed to come from isn't working. I'm getting errors that look like this. 
`access to XMLHttpRequest at 'https://wanderdrone.appspot.com/?_=1681221687324' from origin 'http://www.liedman.net' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.` 
maybe @reqwest should work on it? https://github.com/ded/reqwest

**Comment by perliedman** [comment]
Hi and thanks for reporting. Yeah, the data source used in the demo has unfortunately gone away. I should do something about it, but haven't found the time/energy.

Duplicate of #184.
